pkgs/applications/misc/tty-share/default.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, buildGoPackage, fetchgit }:
+
+# Upstream has a `./vendor` directory with all deps which we rely upon.
+buildGoPackage rec {
+  name = "tty-share";
```

`name` is differed automatically

```suggestion
  pname = "tty-share";
```